### DISCUSSION TITLE
Upcoming Features and Enhancements

### DISCUSSION THREAD

**Discussion by binokochumolvarghese** [binokochumolvarghese/lightbi-hugo#75] [Announcements]
### [Demo site](https://lightbi-hugo-theme.netlify.app/)  

-------------

This list highlights planned updates and new features for the theme. Have feature ideas? Feel free to share them—I’ll aim to integrate them as time permits!

Checkout the [beta demo site](https://lightbi-beta-demo.netlify.app/en/) for the upcoming features.

 - Dynamic Image Gallery
Add a customizable image gallery with a responsive layout, lightbox functionality, and various display options.

  -  Implement multiple theme modes, such as: 
  - Profile Mode for personal pages or portfolios
  - Blog Mode optimized for content-heavy sites
 
- Enhanced Author Section
Revamp the author section in posts to allow for richer information and styling options, such as bio text, social links, and profile images.

**Comment by jfmatth** [comment]
Ability to change homepage or start page.

**Reply by binokochumolvarghese** [reply]
Thanks for the suggestion. Will try to implement that.

**Reply by binokochumolvarghese** [reply]
H, the above have been updated with profile mode for the theme.

**Comment by Queen-Kelda** [comment]
Hi, Love the theme, but your beta version looks fabulous, and would suit my site better.  Any ideas as to when you might release the beta version please?
Thanks for your hard work by the way

**Comment by binokochumolvarghese** [comment]
Hi, Thank you for the kind words. I hope I can release it soon, mostly this month itself.

**Reply by Queen-Kelda** [reply]
Fabulous, Can't wait

**Reply by Queen-Kelda** [reply]
Hi again,  I've just been wondering if it would be possible for you to maybe add a way to pin a post on the home page?  And maybe a "back to the top" button? Once I've got my website up and running I'll send you the link.  Thanks once again

**Reply by binokochumolvarghese** [reply]
@Queen-Kelda : I have updated the version 2 of Lightbi. Please check and let me know your comments. I will work on the additional features soon.

**Reply by Queen-Kelda** [reply]
Hi @binokochumolvarghese Bino, (if I may call you by name) 
I've just updated using Lightbi version 2 and I have to say I love it.  I'm just about to upload it to my website, and I'm really over the moon with it.  Thank you for all your hard work.

**Reply by binokochumolvarghese** [reply]
Hi! Thank you for your kind words — I’m really glad you liked the new version. Can’t wait to see your new website!

**Reply by Queen-Kelda** [reply]
Hi again. I've uploaded now 
https://www.fourbranches.co.uk/blog/index.html 
and thank you

**Reply by binokochumolvarghese** [reply]
I'm glad my theme was helpful for your family site.

**Comment by Queen-Kelda** [comment]
Is there any way to add a dropdown menu please?

**Reply by binokochumolvarghese** [reply]
Yes it there. Check the demo please
